ecs-init/docker/dependencies_test.go

```diff
@@ -32,27 +34,19 @@ var netError = &url.Error{Err: &net.OpError{Op: "read", Net: "unix", Err: io.EOF
 var httpError = &docker.Error{Status: http.StatusInternalServerError, Message: "error"}
 
 func TestIsNetworkErrorReturnsTrue(t *testing.T) {
-	if !isNetworkError(netError) {
-		t.Errorf("Expected true when checking if network error")
-	}
+	assert.True(t, isNetworkError(netError), "Expect network error to be true if netError passed in")
 }
 
 func TestIsNetworkErrorReturnsFalse(t *testing.T) {
-	if isNetworkError(fmt.Errorf("error")) {
-		t.Errorf("Expected false when checking if network error")
-	}
+	assert.False(t, isNetworkError(fmt.Errorf("error")), "Expect network error to be false non-network error passed in")
```

Comment:
Same here. "Expect isNetworkError to return false if non-network error passed in"?